README.md

```diff
@@ -1,27 +1,151 @@
 ###Ruby Wrapper to the PlotLy REST API
 
-**Example Usage**
+**Connecting to the API**
 
 ```ruby
 
 plotly = PlotLy.new('username', 'api key')
```

Comment:
Let's also link the Python docs for folks wanting to check out other plots:

"For more plot types, see the [Plotly Python client docs](https://plot.ly/python/). The Python client uses the same REST API as this Ruby client, so syntax for other plot types can be inferred from here.